Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/AdapterBuilder.h

```diff
@@ -0,0 +1,116 @@
+        //! Convenience method, calls EndNode<Nodes::Adapter>()
+        void EndAdapter();
+        //! Convenience method, calls BeginNode<Nodes::Row>()
+        void BeginRow();
+        //! Convenience method, calls EndNode<Nodes::Row>()
+        void EndRow();
+        //! Convenience method, calls BeginNode<Nodes::Label>()
+        void BeginLabel();
+        //! Convenience method, calls EndNode<Nodes::Label>()
+        void EndLabel();
+        //! Convenience method, calls BeginNode<Nodes::PropertyEditor>()
+        //! If type and value are specified, the Nodes::PropertyEditor::Type attribute and the
+        //! property editor value shall be set.
+        void BeginPropertyEditor(AZStd::string_view type = "", Dom::Value value = {});
+        //! Convenience method, calls EndNode<Nodes::PropertyEditor>()
+        void EndPropertyEditor();
+
+        //! Inserts a Label node with the specified text.
+        void Label(AZStd::string_view text, bool copy = true);
+
+        //! Sets the value of the last node. Used for setting the current value of a property editor.
+        void Value(Dom::Value value);
+        //! Sets an attribute of the last node. Rows, labels, and property editors all support different attributes.
+        //! \see DocumentPropertyEditor::Nodes
+        void Attribute(Name attribute, Dom::Value value);
+
+        //! Returns true if an error has been encountered during the build process, 
+        bool IsError() const;
+        //! Returns the error information, if any, encountered during the build process.
+        const AZStd::string& GetError() const;
+        //! Ends the build operation and retrieves the builder result.
+        //! Operations are no longer valid on this builder once this is called.
+        Dom::Value&& FinishAndTakeResult();
+
+        template <class NodeDefinition>
+        void BeginNode()
+        {
+            BeginNode(GetNodeName<NodeDefinition>());
+        }
+
+        template <class NodeDefinition>
+        void EndNode()
+        {
+            EndNode(GetNodeName<NodeDefinition>());
+        }
+
+        template <class AttributeType>
+        void Attribute(const AttributeDefinition<AttributeType>& definition, Dom::Value value)
+        {
+            Attribute(definition.GetName(), Dom::Value(AZStd::move(value)));
+        }
+
+        template <class AttributeType>
+        void Attribute(const AttributeDefinition<AttributeType>& definition, AttributeType value)
+        {
+            Attribute(definition.GetName(), definition.ValueToDom(AZStd::move(value)));
+        }
+
+        template <>
+        void Attribute<AZStd::string_view>(const AttributeDefinition<AZStd::string_view>& definition, AZStd::string_view value)
+        {
+            Attribute(definition.GetName(), Dom::Value(value, true));
+        }
+
+
+    private:
+        void Error(AZStd::string_view message);
+        Dom::Value& CurrentNode();
+
+        RoutingAdapter* m_routingAdapter = nullptr;
```

Comment:
I couldn't find any classes about RoutingAdapter. Is this being added in anticipation of a future change? If so, this can probably be deleted unless you are absolutely sure that this will be needed.

Reply:
Oh yep just bad git on my end leaving a tiny bit of that old change in.